apps/giftify/src/app/list/list.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { Gift } from '../shared/gift.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ListService {
+  giftList: Gift[] = [
```

Comment:
To może być prywatne
```suggestion
  private giftList: Gift[] = [
```